src/components/hotjarTrigger/HotjarTrigger.js

```diff
@@ -21,18 +21,26 @@ HotjarTrigger.propTypes = {
     children: node.isRequired,
 };
 
-export const LandingssideMedSakerFraInnsynHotjarTrigger = ({children}) => (
-    <HotjarTrigger hotjarTrigger="landingsside_med_saker_fra_innsyn">{children}</HotjarTrigger>
+export const SoknadMedInnsynHotjarTrigger = ({children}) => (
+    <HotjarTrigger hotjarTrigger="soknad_med_innsyn">{children}</HotjarTrigger>
 );
 
-LandingssideMedSakerFraInnsynHotjarTrigger.propTypes = {
+SoknadMedInnsynHotjarTrigger.propTypes = {
     children: node,
 };
 
-export const LandingssideUtenSakerFraInnsynHotjarTrigger = ({children}) => (
-    <HotjarTrigger hotjarTrigger="landingsside_uten_saker_fra_innsyn">{children}</HotjarTrigger>
+export const SoknadMedInnsynFraBergenHotjarTrigger = ({children}) => (
+    <HotjarTrigger hotjarTrigger="soknad_med_innsyn_fra_bergen">{children}</HotjarTrigger>
 );
 
-LandingssideUtenSakerFraInnsynHotjarTrigger.propTypes = {
+SoknadMedInnsynFraBergenHotjarTrigger.propTypes = {
```

Comment:
Denne ser ikke ut som om den er i bruk.

Reply:
Det var den jeg mente her: Klargjør også en egen trigger for Bergen, som vi ønsker å ha i overgangen når vi går fra å kun ha Bergen på innsyn til å også få nye kommuner. Må vente på at backend sender kommunenummer i /kommune først

Men har fått ut backendfiksen så jeg inkluderer den loggen nå ^^